code/modules/clothing/under/f13.dm

```diff
@@ -1844,3 +1844,45 @@
 	icon_state = "bibledog"
 	item_state = "bibledog"
 	armor = list("melee" = 5, "bullet" = 5, "laser" = 2, "energy" = 2, "bomb" = 5, "bio" = 0, "rad" = 10, "fire" = 50, "acid" = 40)
+
+
+
+/*
+///////////////////////////////////////////
+// This is used for Badlands Content.     /
+// Primarily, the GMB, DFS and LBJ groups./
+///////////////////////////////////////////
+*/
+//GMB
+/obj/item/clothing/under/f13/ranger/modif_ranger/gmb
+	name = "surplus militia outfit"
+	desc = "A surplus outfit with a green cotton longshirt and dark grey jeans along with a black bandana around the neck. It has been fitted with the militia's patch."
+	body_parts_covered = CHEST|GROIN|LEGS|ARMS
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 30, "fire" = 20, "acid" = 50)
+
+/obj/item/clothing/under/f13/rustic/gmb
+	name = "rustic militia outfit"
+	desc = "Simple rustic clothes for your day to day life in the wastes, fitted with the militia's patch."
+	body_parts_covered = CHEST|GROIN|LEGS|ARMS
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 30, "fire" = 20, "acid" = 50)
+
+//DFS
+/obj/item/clothing/under/f13/merca/dfs
+	name = "security team outfit"
+	desc = "A well put together uniform, alongside the DFS insignia."
+	body_parts_covered = CHEST|GROIN|LEGS|ARMS
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 30, "fire" = 20, "acid" = 50)
+
+/obj/item/clothing/under/f13/mercc/dfs
+	name = "security lead outfit"
+	desc = "A well put together uniform, alongside the DFS insignia."
+	body_parts_covered = CHEST|GROIN|LEGS|ARMS
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 30, "fire" = 20, "acid" = 50)
```

Comment:
Must you use two almost identical paths? I'd rather prefer the paths expanded to be more different than a single letter that is not even a subtype :X

Reply:
Blame the original coders that came up with the horrid paths as is. I'd rather it wasn't that way, but it's how it is.